ccdproc/image_collection.py

```diff
@@ -69,6 +69,7 @@ def __init__(self, location=None, keywords=None, info_file=None,
         self._location = location
         self._filenames = filenames
         self._files = []
+        self._info_file = info_file
```

Comment:
Is it really necessary to keep the info-file as attribute? I'm not sure but couldn't that interfere in the representation with the other attributes?